pallets/eth-bridge/src/lib.rs

```diff
@@ -709,6 +709,24 @@ pub mod pallet {
                     } else {
                         InvalidTransaction::Custom(3u8).into()
                     },
+                Call::submit_discovered_events { author, range, events_fraction, signature } =>
```

Comment:
I missed the other PR but does it make sense to rename the extrinsic to something like `submit_ethereum_events`? It feels more descriptive and helps understand exactly what this extrinsic does.

Reply:
can do